### DISCUSSION TITLE
More accesible keymap

### DISCUSSION THREAD

**Discussion by CrumblyLiquid** [monkeytypegame/monkeytype#1218] [Ideas]
From what I've experienced and seend there's sometimes counfusion about the keymap. You enable it in the "Change keymap mode" which is **confuisng** to some people.

I think it would be nice to split it into **two settings**:
- **Toggle keymap** (on/off)
- **Change keymap mode** (static/next/react)

What do you guys think?

**Comment by frogfrogfrogfrogfrog** [comment]
This seems like a niche idea. It still makes sense to put all the settings under the "change keymap mode". You can somewhat solve this by enabling "single list command line" in which it doesn't force you to enable it under "change keymap mode" and you can see all the options without the confusion.

**Reply by unrealapex** [reply]
I agree, but it would make the settings for the keymap be more accessible.

**Reply by frogfrogfrogfrogfrog** [reply]
I still don't quite understand. If your definiton of "accessible" being that you can access something easier/faster than the solution would still work correct?

**Reply by CrumblyLiquid** [reply]
My idea of more accessible is that it should be easier for people (especially new people) to find. I don't think how fast you can access it is a problem since you don't use this setting often.

It would be simply easier for especially new people to find. Enabling single line commands might make it easier but new people probably won't know it's an option.
